packages/typography/vitest.config.ts

```diff
@@ -0,0 +1,11 @@
+import {defineConfig} from "vitest/config";
+
+export default defineConfig({
+  test: {
+    // use JS DOM for browser-like test environment
+    environment: "jsdom",
```

Comment:
I think we may only need jsdom in the packages that actually have test files that require the browser. Will see about removing this in this and similar packages

Reply:
seems to have worked with no adverse effects